static/app/views/settings/account/notifications/utils.tsx

```diff
@@ -45,398 +32,6 @@ export const groupByOrganization = (
   }, {});
 };
 
-export const getFallBackValue = (notificationType: string): string => {
-  switch (notificationType) {
-    case 'alerts':
-      return 'always';
-    case 'deploy':
-      return 'committed_only';
-    case 'workflow':
-      return 'subscribe_only';
-    case 'approval':
-      return 'always';
-    case 'quota':
-      return 'always';
-    case 'spikeProtection':
-      return 'always';
-    case 'reports':
-      return 'always';
-    default:
-      // These are the expected potential settings with fallback of ''
-      // issue, quotaErrors, quotaTransactions, quotaAttachments,
-      // quotaReplays, quotaWarnings, quotaSpendAllocations
-      return '';
-  }
-};
-
-export const providerListToString = (providers: string[]): string => {
-  return providers.sort().join('+');
-};
-
-export const getChoiceString = (choices: string[][], key: string): string => {
-  if (!choices) {
-    return 'default';
-  }
-  const found = choices.find(row => row[0] === key);
-  if (!found) {
-    throw new Error(`Could not find ${key}`);
-  }
-
-  return found[1];
-};
-
-const isDataAllNever = (data: {[key: string]: string}): boolean =>
-  !!Object.keys(data).length && Object.values(data).every(value => value === 'never');
-
-const getNonNeverValue = (data: {[key: string]: string}): string | null =>
```

Comment:
@scttcper it's crazy how many mapping functions we had in this file to deal with the fact our data model sucks and it feels very nice to delete all this crap as we don't need anything like this anymore